### ISSUE TITLE
Klassifizierungen exportieren und klonen

### ISSUE THREAD

**Issue by buchen** [buchen/portfolio#374]
Aus dem Wertpapier Forum:

> PP sollte mehr Klassifizierungsmuster mitliefern. Die fünf mitgelieferten Muster decken nicht allen Bedarf ab. Natürlich kann jeder sich seine Vorlagen selbst erstellen. Neben einer Anbieterklassifizierung habe ich mir eine mehrschichtige Anlageklassen-Klassifizierung erstellt - und bin auch von ihr überzeugt. Aber sie ist halt im stillen Kämmerchen ohne Feedback Anderer entstanden. Deshalb bleibt ein ungutes Gefühl. Es wäre toll, wenn Leute ihre Klassifizierungen teilen könnten - dann könnte man von anderen lernen bzw. sich Arbeit sparen. Eine Export-Option, die alle Wertpapier-Daten löscht, wäre praktisch, denn dann müsste man nicht mehr in der XML einen Taxonomy-Baum ausschneiden. Vielleicht wäre es auch hilfreich, wenn im Kontextmenü einer Taxonomie/Klassifizierung es die Option "klonen" gäbe. Dann könnte man experimentieren, ob eine leichte Modifikation besser ist. Das würde es einem ersparen, zahlreiche neue Versionen der eigenen XML-Datendatei zu erstellen und je wieder zu bestimmten Vorversionen zurückspringen zu müssen. Deshalb habe ich in der xml-Datei eine taxonomy kopiert, darin alle GUIDs ersetzt und das bereinigte dann wieder eingesetzt. (Das scheint zu gehen, ist das eigentlich alles?). Würde ich eine sehr lange Taxonomie anpassen wollen, ginge das wohl nicht mehr manuell und ich müsste nochmal zu sed/grep/regex recherchieren....

Gute Idee.

* Klonen mit Wertpapier Referenz? (ISIN? WKN? Ticker?)
* Könnte man Zuordnung einer konkreten Klassifizierung exportieren (Aero) - was mit den GUIDs?

**Comment by Mogito** [comment]
Hallo, ein Klonen einer Asset-Allocation für verschiedene Portfolios wäre auch sehr hilfreich.

**Comment by buchen** [comment]
@Mogito Du meinst über verschiedene Datei hinweg? Das wäre ja ein "export" (inklusive der Zuweisungen) und dann ein "import" oder?

**Comment by Mogito** [comment]
Nein, ich meinte innerhalb einer Datei. Ich habe mir die AssetAllocation nach Komer aufgebaut, würde diese aber gerner für mehrere Portfolios innerhalb einer Datei nutzen. Dafür habe ich zwei Ansätze:
1. eine Asset Allocation in verschiedene Portfolios unterteilt, und darunter die Aufteilung nach Komer. Dann brauche ich die Substrukturen für jedes Portfolio im Baum.
2. Je eine Allocation pro Portfolio, da würde das Klonen wohl am meisten helfen.

Ließe es sich dann noch einrichten, dass man die Reihenfolge der Asset Allocations in der Liste auf der linken Seite ändern kann?

**Comment by buchen** [comment]
> Ließe es sich dann noch einrichten, dass man die Reihenfolge der Asset Allocations in der Liste auf der linken Seite ändern kann?

Innerhalb einer Datei kann man die Reihenfolge ändern und kopieren (klonen) - einfach mal eine Rechtsklick auf den Eintrag in der Seitennavigation:
<img width="360" alt="bildschirmfoto 2016-01-17 um 12 03 09" src="https://cloud.githubusercontent.com/assets/587976/12377127/5bcf29ae-bd12-11e5-9d12-dc10cde6efa0.png">

**Comment by Mogito** [comment]
Merci, das hatte ich gesucht. Die einfachsten Wege sieht man manchmal nicht. Danke!
